ruby/conditional_modifiers.md

```diff
@@ -0,0 +1,86 @@
+# Limit use of conditional modifiers to short, simple cases
+
+Conditional modifiers (i.e., `if` or `unless` at the end of a line) can be
+surprising when they appear on long or complex lines. The reader might not see
+them while scanning the code.
+
+So, prefer to use them only for short, simple cases. For example:
+
+```ruby
+do_later if async?
+```
+
+The example above can read more naturally than:
+
+```rb
+if async?
+  do_later
+end
+```
+
+## Complex conditions
+
+However, if the line is too long (around 80 characters) or complex (e.g., an
+`if` with multiple conditions like `if a && b`) prefer the multi-line form:
+
+```ruby
+# Avoid
+block_access! if signed_in? && !current_user.active?
+
+# Prefer
+if signed_in? && !current_user.active?
+  block_access!
+end
+```
+
+There might be cases where the conditional modifier work well with multiple
+conditions, so use your best judgment.
+
+## An opportunity to refactor
+
+If the conditions are related, consider extracting a method that groups them.
+This might allow you to use the conditional modifier form again.
+
+```ruby
+def inactive_user?
+  signed_in? && !current_user.active?
+end
+
+block_access! if inactive_user?
+```
```

Comment:
This seems to be moving towards the  [Triangle of Separation](https://thoughtbot.com/blog/triangle-of-separation) style of code